### PR TITLE
Multiple fixes

### PR DESCRIPTION
**Pull request by gmbr3**
* ReallocatePool new ABI
* old CopyMem restored as CopyMemC
* Add more pkgconfig variables
* Switch files to Unix LF 

Fixes ncroxon/gnu-efi#9

### CODE REVIEW

File: gnuefi/Makefile [hunk 1]
```diff
@@ -46,6 +46,34 @@ FILES	= reloc_$(ARCH)
...
+ifeq ($(ARCH),ia64)
+  efi_machine_type_name=ia64
+endif
+ifeq ($(ARCH),riscv64)
+  efi_machine_type_name=riscv64
+endif
+ifeq ($(ARCH),loongarch64)
+  efi_machine_type_name=loongarch64
+endif
+
```
Comment (gmbr3) [inline gnuefi/Makefile:76]: Missing mips

File: lib/Makefile [hunk 2]
```diff
@@ -42,7 +42,7 @@ TOPDIR = $(SRCDIR)/..
 include $(SRCDIR)/../Make.defaults
 
 FILES = boxdraw smbios console crc data debug dpath \
-        entry error event exit guid hand hw init lock \
+        entry error event exit guid hand hw init legacy lock \
```
Comment (gmbr3) [inline lib/Makefile:45]: Will need adding to MSVC ci

File: apps/Makefile [hunk 3]
```diff
@@ -62,7 +62,8 @@ TARGET_APPS = t.efi t2.efi t3.efi t4.efi t5.efi t6.efi \
 	      route80h.efi drv0_use.efi AllocPages.efi exit.efi \
 	      FreePages.efi setjmp.efi debughook.efi debughook.efi.debug \
 	      bltgrid.efi lfbgrid.efi setdbg.efi unsetdbg.efi \
-	      ctors_test.efi ctors_dtors_priority_test.efi
+	      ctors_test.efi ctors_dtors_priority_test.efi \
+	      old_ABI.efi
```
Comment (gmbr3) [inline apps/Makefile:66]: Needs rebase

**Comment by ncroxon** [comment]
Callum, can you please sign your patches.

**Comment by gmbr3** [comment]
> Callum, can you please sign your patches.

Ah, I missed one

File: gnuefi/Makefile (hunk 1 above)
Comment (gmbr3) [reply gnuefi/Makefile:79]: done

**Comment by gmbr3** [comment]
> EDIT: I presume you mean the Signed-off-by tag, they are already GPG signed

Added the missing Signed-off-by tag

**Comment by gmbr3** [comment]
Forgot to add Bug References

**Review by ncroxon — APPROVED** [review]

**Pull request merged**